include/swift/Parse/SyntaxParsingContext.h

```diff
@@ -363,7 +363,7 @@ class alignas(1 << SyntaxAlignInBits) SyntaxParsingContext {
   /// This function will be called during the destroying of a root syntax
   /// parsing context. However, we can explicitly call this function to get
   /// the syntax tree before closing the root context.
-  void finalizeRoot();
+  RC<RawSyntax> finalizeRoot();
```

Comment:
Do you have plan to use this result?

Reply:
Yes, my intention is for parsing to return the root raw node instead of directly modifying SourceFile.